examples/preprocessing/eeg_bridging.py

```diff
@@ -0,0 +1,255 @@
+for sub in range(1, 11):
+    print(f'Computing electrode bridges for subject {sub}')
+    raw = mne.io.read_raw(mne.datasets.eegbci.load_data(
+        subject=sub, runs=(1,))[0], preload=True, verbose=False)
+    mne.datasets.eegbci.standardize(raw)  # set channel names
+    montage = mne.channels.make_standard_montage('standard_1005')
+    raw.set_montage(montage, verbose=False)
+    raw_data[sub] = raw
+    ed_data[sub] = mne.preprocessing.compute_bridged_electrodes(raw)
+
+
+# %%
+# Examine an Electrical Distance Matrix
+# -------------------------------------
+# Before we look at the electrical distance distributions across subjects,
+# let's look at the distance matrix for one subject and try and understand
+# how the algorithm works. We'll use subject 6 as it is a good example of
+# bridging. In the zoomed out color scale version on the right, we can see
+# that there is a distribution of electrical distances that are specific to
+# that subject's head physiology/geometry and brain activity during the
+# recording. On the right, when we zoom in, we can see several electrical
+# distance outliers that are near zero; these indicate bridging.
+
+bridged_idx, ed_matrix = ed_data[6]
+fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(8, 4))
+fig.suptitle('Subject 6 Electrical Distance Matrix')
+# take median across epochs, only use upper triangular, lower is NaNs
+ed_plot = np.zeros(ed_matrix.shape[1:]) * np.nan
+triu_idx = np.triu_indices(ed_plot.shape[0], 1)
+for idx0, idx1 in np.array(triu_idx).T:
+    ed_plot[idx0, idx1] = np.nanmedian(ed_matrix[:, idx0, idx1])
+im1 = ax1.imshow(ed_plot, aspect='auto')
+cax1 = fig.colorbar(im1, ax=ax1)
+cax1.set_label(r'Electrical Distance ($\mu$$V^2$)')
+# zoomed in colors
+im2 = ax2.imshow(ed_plot, aspect='auto', vmax=5)
+cax2 = fig.colorbar(im2, ax=ax2)
+cax2.set_label(r'Electrical Distance ($\mu$$V^2$)')
+for ax in (ax1, ax2):
+    ax.set_xlabel('Channel Index')
+    ax.set_ylabel('Channel Index')
+fig.tight_layout()
+
+# %%
+# Examine the Distribution of Electrical Distances
+# ------------------------------------------------
+# Now let's plot a histogram of the electrical distance matrix. Note that the
+# electrical distance matrix is upper triangular but does not include the
+# diagonal from the previous plot. This means that the pairwise electrical
+# distances are not computed between the same channel (which makes sense as
+# the differences between a channel and itself would just be zero). The initial
+# peak near zero therefore represents pairs of different channels that
+# are nearly identical which is indicative of bridging. EEG recordings
+# without bridged electrodes do not have a peak near zero.
+
+fig, ax = plt.subplots(figsize=(5, 5))
+fig.suptitle('Subject 6 Electrical Distance Matrix Distribution')
+ax.hist(ed_matrix[~np.isnan(ed_matrix)], bins=np.linspace(0, 500, 51))
+ax.set_xlabel(r'Electrical Distance ($\mu$$V^2$)')
+ax.set_ylabel('Count')
```

Comment:
it seems that this counts all channel paris * epochs, right? If so, it would be good to clarify this somhow, as it may be surprising to see count > 200.